lib/threema/receive/file.rb

```diff
@@ -26,6 +26,7 @@ def initialize(content:, threema:, **)
 
         @mime_type = structure['m']
         @name      = structure['n']
+        @caption = structure['d']
```

Comment:
again some weird white space, otherwise looks good!

Reply:
As discussed, I've followed the pattern established in the gem and maybe we can look into the rubocop settings.